.gitlab-ci.yml

```diff
@@ -61,9 +62,7 @@ check-std:
   stage:                           check
   <<:                              *docker-env
   script:
-    - for crate in ${ALL_CRATES}; do
-        cargo check --verbose --all-features;
-      done
+    - cargo check --verbose --all-features;
```

Comment:
Missing the `--all` flag if you want to go without the loop ... however, I really do not recommend doing so. Why have you changed it back?